### PR TITLE
Convert remaining institutionId references and uses of institution.id

### LINKED ISSUE #1103: Refactor change all references to institution id's to lei
https://github.com/cfpb/hmda-platform-ui/pull/1102 introduced some weirdness where we're referring to an "institution id" in different ways; sometimes `lei`, sometimes `id`, and sometimes `institutionId`.

[This comment](https://github.com/cfpb/hmda-platform-ui/pull/1102#issuecomment-434763053) sums up the concern and the reason this issue was created.

### PR DESCRIPTION
**Pull request by wpears**
Closes #1103

### CODE REVIEW

File: src/actions/fetchCSV.js [hunk 1]
```diff
@@ -6,11 +6,11 @@ import { getCSV } from '../api/api.js'
 import { error } from '../utils/log.js'
 
 // downloading the csv edit reports, no reducer required
-export default function fetchCSV(institutionId, filing, submissionId) {
+export default function fetchCSV(lei, filing, submissionId) {
   return dispatch => {
     dispatch(requestCSV())
     return getCSV({
-      id: institutionId,
+      id: lei,
```
Comment (awolfe76) [inline src/actions/fetchCSV.js]: Should we pass this as `lei: lei` too?

If so, we'd need to update [`makeUrl`](https://github.com/cfpb/hmda-platform-ui/blob/393ab2cc9497be3e5345f3) too. And `makeUrl` has some [commented out `env` stuff](https://github.com/cfpb/hmda-platform-ui/blob/393ab2cc9497be3e5345f328eb99f9f74ed61f7b/src/api/makeUrl.js#L2-L6) in it that we can remove.

File: src/actions/fetchIRSCSV.js [hunk 2]
```diff
@@ -6,11 +6,11 @@ import { getIRSCSV } from '../api/api.js'
 import { error } from '../utils/log.js'
 
 // downloading the IRS csv, no reducer required
-export default function fetchIRSCSV(institutionId, filing, submissionId) {
+export default function fetchIRSCSV(lei, filing, submissionId) {
   return dispatch => {
     dispatch(requestIRSCSV())
     return getIRSCSV({
-      id: institutionId,
+      id: lei,
```
Comment (awolfe76) [inline src/actions/fetchIRSCSV.js]: Same as above, `id -> lei`.

File: src/actions/fetchNewSubmission.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ import { error } from '../utils/log.js'
 export default function fetchNewSubmission() {
   return (dispatch, getState) => {
     const appState = getState().app
-    const id = appState.institutionId
+    const id = appState.lei
```
Comment (awolfe76) [inline src/actions/fetchNewSubmission.js]: `const lei` here too.

Comment (awolfe76) [reply src/actions/fetchNewSubmission.js]: And if so, [line 18](https://github.com/cfpb/hmda-platform-ui/pull/1204/files#diff-22717cca12596cbafadf42630682809bR18) 👇 would need to be updated.

File: src/actions/processFileErrors.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ import receiveFileErrors from './receiveFileErrors.js'
 
 export default function processFileErrors(errors, file) {
   return (dispatch, getState) => {
-    const id = getState().app.institutionId
+    const id = getState().app.lei
```
Comment (awolfe76) [inline src/actions/processFileErrors.js]: `id -> lei`

File: src/actions/requestUpload.js [hunk 5]
```diff
@@ -4,7 +4,7 @@ export default function requestUpload() {
   return (dispatch, getState) => {
     return dispatch({
       type: types.REQUEST_UPLOAD,
-      id: getState().app.institutionId
+      id: getState().app.lei
```
Comment (awolfe76) [inline src/actions/requestUpload.js]: Same here, but then the upload reducer would need updated too.

File: src/actions/selectFile.js [hunk 6]
```diff
@@ -3,7 +3,7 @@ import suppressEdits from './suppressEdits.js'
 
 export default function selectFile(file) {
   return (dispatch, getState) => {
-    const id = getState().app.institutionId
+    const id = getState().app.lei
```
Comment (awolfe76) [inline src/actions/selectFile.js]: `lei -> id`

File: src/actions/selectNewFile.js [hunk 7]
```diff
@@ -5,7 +5,7 @@ export default function selectNewFile(file) {
     return dispatch({
       type: types.SELECT_NEW_FILE,
       file,
-      id: getState().app.institutionId
+      id: getState().app.lei
```
Comment (awolfe76) [inline src/actions/selectNewFile.js]: `id -> lei`

File: src/api/fetch.js [hunk 8]
```diff
@@ -15,7 +15,7 @@ export function setStore(s) {
 export function getFilingData() {
   const appState = store.getState().app
   return {
-    id: appState.institutionId,
+    id: appState.lei,
```
Comment (awolfe76) [inline src/api/fetch.js]: `id -> lei`

File: src/modals/confirmationModal/container.jsx [hunk 9]
```diff
@@ -10,7 +10,7 @@ import checkFileErrors from '../../utils/checkFileErrors.js'
 import ConfirmationModal from './index.jsx'
 
 export function mapStateToProps(state) {
-  const id = state.app.institutionId
+  const id = state.app.lei
```
Comment (awolfe76) [inline src/modals/confirmationModal/container.jsx]: `id -> lei`

File: src/submission/upload/container.jsx [hunk 10]
```diff
@@ -4,7 +4,7 @@ import handleFile from '../../actions/handleFile.js'
 import pollForProgress from '../../actions/pollForProgress.js'
 
 export function mapStateToProps(state) {
-  const id = state.app.institutionId
+  const id = state.app.lei
```
Comment (awolfe76) [inline src/submission/upload/container.jsx]: `id -> lei`

**Review by awolfe76 — CHANGES_REQUESTED** [review]
I think there are just a few more places where the `id -> lei` switch would help.

**Comment by wpears** [comment]
:+1:

File: src/modals/confirmationModal/container.jsx [hunk 11]
```diff
@@ -10,15 +10,15 @@ import checkFileErrors from '../../utils/checkFileErrors.js'
...
   const { showing } = state.app.confirmation
   const { filingPeriod } = state.app
   const { code } = state.app.submission.status
   const { newFile } =
-    state.app.upload[id] || state.app.upload['__DEFAULT_UPLOAD__']
+    state.app.upload[lei] || state.app.upload['__DEFAULT_UPLOAD__']
 
   return {
-    id,
+    lei,
```
Comment (awolfe76) [inline src/modals/confirmationModal/container.jsx:21]: This `lei` update requires changes to [`confirmationModal/index.jsx`](https://github.com/cfpb/hmda-platform-ui/blob/393ab2cc9497be3e5345f328eb99f9f74ed61f7b/src/modals/confirmationModal/index.jsx#L49) too.

Comment (awolfe76) [reply src/modals/confirmationModal/container.jsx:21]: Its causing

```
TypeError: Cannot read property 'focus' of undefined
ModalConfirm._focusButton
src/modals/confirmationModal/index.jsx:8
   5 | import '../Modal.css'
   6 | 
   7 | let _focusButton = function() {
>  8 |   return this.confirmButton.focus()
   9 | }
  10 | 
  11 | let _focusLink = function() {
```

**Review by awolfe76 — APPROVED** [review]

**Pull request merged**